doc/topics/releases/releasecandidate.rst

```diff
@@ -2,53 +2,89 @@
 
 .. _release-candidate:
 
-===========================================
-Installing/Testing a Salt Release Candidate
-===========================================
+=====================================
+Install/Test a Salt Release Candidate
+=====================================
 
 When it's time for a new feature release of Salt, follow the instructions below to
 install the latest release candidate of Salt, and try all the shiny new
 features! Be sure to report any bugs you find on `Github
 <https://github.com/saltstack/salt/issues/new/>`_.
 
-Installing Using Packages
-=========================
 
+Install using packages
+======================
 Builds for a few platforms are available as part of the RC at https://repo.saltproject.io/salt_rc/.
-The builds should include the latest version of the OS that is currently available.
+The builds should include the latest version of the operating system that is currently available.
 
 .. note::
 
-    Follow the instructions on https://repo.saltproject.io/,
-    but insert ``salt_rc/`` into the URL between the hostname
-    and the remainder of the path.
+   Beginning with the 3005 (Phosphorus) release of Salt, the Salt Project is
+   changing its packaging system to Tiamat. Any new operating systems added in 3005
+   will only have Tiamat packages. The Salt Project will phase out the old Salt
+   package builds for currently supported operating systems by 3007. See
+   `What is Tiamat? <https://docs.saltproject.io/salt/install-guide/en/latest/topics/upgrade-to-tiamat.html#what-is-tiamat>`_
+   for more information.
 
-    For Redhat Python 3
+To install release candidate packages:
 
-    .. code-block:: bash
+Follow the instructions for your operating system on https://repo.saltproject.io/,
+but insert ``salt_rc/`` into the URL between the hostname and the remainder
+of the file path.
 
-        baseurl=https://repo.saltproject.io/salt_rc/py3/redhat/$releasever/$basearch/
 
-    For Ubuntu Python 3 (replace os_version, with ubuntu version. For example 20.04)
+Tiamat packages
+---------------
+For RedHat:
 
-    .. code-block:: none
+.. code-block:: bash
 
-        deb https://repo.saltproject.io/salt_rc/py3/ubuntu/<os_version>/amd64 focal main
+    baseurl=https://repo.saltproject.io/salt_rc/salt/py3/redhat/$releasever/$basearch/
 
-    For Debian Python 3 (replace os_version, with debian version. For example 10)
+For Ubuntu, replace the ``<os_version>`` variable with number of the Ubuntu
+version. For example ``20.04``:
```

Comment:
would add something along the lines 
```
and ensure the codename matches the os version , for example 20.04 is focal
```

Reply:
I took this suggestion, but made it 22.04 and Jammy to reflect the latest.